src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -47,7 +47,7 @@ def load_arguments(self, _):
             c.argument('timeout', options_list=['--timeout'], type=int, default=3600,
                        help='Time in seconds for the copy operation to finish. Increase this time if '
                        'you are going to copy large images (disks) like 512GB or more.')
-            c.argument('temporary_resource_group_name', options_list=['--temporary_resource_group_name'],
+            c.argument('temporary_resource_group_name', options_list=['--temporary-resource-group-name'],
```

Comment:
This should be a break change, right ?

Reply:
Yes. The old name is wrong. We should use "-" instead of "_".

Reply:
Is it possible to add `--temporary-resource-group-name` while deprecating `--temporary_resource_group_name`? Then you can remove `--temporary_resource_group_name` after a while.

Reply:
I support both now.